Cargo.toml

```diff
@@ -118,3 +118,4 @@ codegen-units = 4
 lto = true
 opt-level = 3
 debug = true
+codegen-units = 2
```

Comment:
can it reduce the performance?

Reply:
Maybe, but at least it won't OOM (on my machine and @Hoverbear's laptop) anymore.

Reply:
It increases compile time, but I do not think performance of the end binary.

Reply:
Could you add a TODO since it's a temporary work around?